Update community page (#177)
tikv/website: content/community/community.md

description: The TiKV community
---

TiKV is an open source project with an active community. TiKV's source code is hosted on [GitHub](https://github.com/tikv/tikv) and is a [Cloud Native Computing Foundation](https://www.cncf.io) incubating project. We have information about how to contribute [here](/community/contribute/).

## SIGs

> The TiKV community is organized into special interest groups (SIGs). More information about the community and its governance is in the [community repository](https://github.com/tikv/community). The current roster of SIGs is listed below. follow the links for more information.

Reviewer: "The current roster of SIGs is listed below." This is not quite right, prefer either "The current roster of SIGs is below." or "The current SIGs are listed below."

* [Coprocessor](/community/sig-coprocessor/)
* [Engine](/community/sig-engine/)
* [Raft](/community/sig-raft/)
* [Transaction](/community/sig-transaction/)

## Communications

> In TiKV community we value open communication and open collaboration. Here are the general communication channels we use to communicate.

Reviewer: ```suggestion
In the TiKV community we value open communication and collaboration. Here are our general communication channels.
```

- **General contact methods**

    - [Slack](https://join.slack.com/t/tikv-wg/shared_invite/enQtNTUyODE4ODU2MzI0LWVlMWMzMDkyNWE5ZjY1ODAzMWUwZGVhNGNhYTc3MzJhYWE0Y2FjYjliYzY1OWJlYTc4OWVjZWM1NDkwN2QxNDE) (Recommended)
    - [Twitter](https://twitter.com/tikvproject)

>     - [Reddit](https://www.reddit.com/r/TiKV)

Reviewer: We should not promote Reddit - the forum is basically unused and Reddit has a poor reputation

    - [Stack Overflow](https://stackoverflow.com/questions/tagged/tikv)

- **Report security vulnerabilities**

    To report a security vulnerability, please send an email to [TiKV-security](tikv-security@lists.cncf.io) group.

- **Community monthly meeting**

  The TiKV team meets on the 4th Wednesday of every month (unless otherwise specified) at 07.00 p.m. PST. See [Meeting notes](https://docs.google.com/document/d/1CWUAkBrcm9KPclAu8fWHZzByZ0yhsQdRggnEdqtRMQ8/edit) for details.

> ## Projects built upon TiKV

Reviewer: ```suggestion
## Projects built on TiKV
```

> As an open-source and vendor-neutral project, TiKV thrives by collaborating with the wider community. Here are are some projects that are built upon TiKV:

Reviewer: ```suggestion
TiKV is an open-source and vendor-neutral project, it thrives by collaborating with the wider community. Here are are some projects that are built using TiKV:
```

- [Tidis](https://github.com/yongman/tidis) -  a distributed NoSQL database, providing a Redis protocol API (string, list, hash, set, sorted set)
- [Titan](https://github.com/meitu/titan) - a distributed implementation of Redis compatible layer based on TiKV

> - [Titea](https://github.com/gengmei-tech/titea) - a distributed redis protocol compatible NoSQL Database, providing a Redis protocol, base on Tikv and PD

Reviewer: ```suggestion
- [Titea](https://github.com/gengmei-tech/titea) - a distributed redis protocol compatible NoSQL Database, providing a Redis protocol, based on Tikv and PD
```

> - [TiPrometheus](https://github.com/bragfoo/TiPrometheus) - A adapter/gateway for TiKV

Reviewer: ```suggestion
- [TiPrometheus](https://github.com/bragfoo/TiPrometheus) - An adapter/gateway for TiKV
```

Should this description mention Prmetheus or data collection or something?

- [TiBigData](https://github.com/pingcap-incubator/TiBigData) - Misc BigData components for TiDB

> - [Zetta](https://github.com/zhihu/zetta) - an open-source NoSQL database that supports Transaction and Cloud Spanner like API

Reviewer: ```suggestion
- [Zetta](https://github.com/zhihu/zetta) - an open-source NoSQL database that supports transactions and a Cloud Spanner-like API
```
